src/VolumeDrawable.ts

```diff
@@ -385,8 +414,8 @@ export default class VolumeDrawable {
         this.pathTracedVolume.updateActiveChannels(this);
       }
     } else {
-      if (this.rayMarchedAtlasVolume) {
-        this.rayMarchedAtlasVolume.fuse(this.fusion, this.volume.channels);
+      if (this.volumeRendering instanceof RayMarchedAtlasVolume) {
+        this.volumeRendering.fuse(this.fusion, this.volume.channels);
```

Comment:
I believe in the upcoming refactor, `fuse` can just do a single call on this.volumeRendering and let each Impl take care of the details :)